### PR TITLE
Add SPM support

### PR DESCRIPTION
**Pull request by pomozoff**
**Description**:
Add SPM support

**Related issue(s)**: #358

### CODE REVIEW

**Comment by mpodeszwa** [comment]
Hi @sbarow , could you please take a look at this PR? 🙂

**Comment by tinder-maxwellelliott** [comment]
@Leland-Takamine Can we land this to get Travis working again?

**Comment by tinder-maxwellelliott** [comment]
Welp gotta get their attention somehow https://twitter.com/elliomax/status/1250814239846961153

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        .target(
+            name: "RIBs",
+            dependencies: ["RxSwift", "RxRelay"],
+            path: "ios/RIBs"
+        ),
+        .testTarget(
+            name: "RIBsTests",
+            dependencies: ["RIBs"],
+            path: "ios/RIBsTests",
+            exclude: ["ComponentizedBuilderTests.swift", "MultiStageComponentizedBuilderTests.swift"]
```
Comment (kovpas) [inline Package.swift:25]: Why did you exclude these two?

**Comment by kovpas** [comment]
Hey @pomozoff, thanks for contribution and sorry for the delay! I've added a comment, please have a look.

File: Package.swift (hunk 1 above)
Comment (pomozoff) [reply Package.swift:25]: Unfortunately, I don't remember. I've added them back and seems like everything builds well.

File: .travis.yml [hunk 2]
```diff
@@ -17,23 +18,26 @@ matrix:
...
+      os: osx
+      osx_image: xcode11.2
       env:
-        global:
         - LC_CTYPE=en_US.UTF-8
         - LANG=en_US.UTF-8
-        - IOS_SDK=iphonesimulator12.2
-        - IOS_VER=12.2 # Tests must run at the same OS version as $IOS_SDK
+        - IOS_SDK=iphonesimulator13.2
+        - IOS_VER=13.2 # Tests must run at the same OS version as $IOS_SDK
```
Comment (kovpas) [inline .travis.yml:35]: This one should be `13.2.2` in order for build to pass, I believe

**Comment by kovpas** [comment]
Thanks @pomozoff. Could you please revert the changes to the templates though? Let's keep this one out of this PR.
Also, travis is failing, I believe because of the iOS simulator version - 13.2 is not available there anymore.

**Comment by kovpas** [comment]
Thanks @pomozoff. Mind also reverting the changes to the templates?

**Comment by pomozoff** [comment]
@kovpas, oops, haven't seen my local changes are in the master, reverted them back

File: .travis.yml [hunk 3]
```diff
@@ -17,23 +18,27 @@ matrix:
...
-      osx_image: xcode10.2
+      os: osx
+      osx_image: xcode11.2
       env:
-        global:
         - LC_CTYPE=en_US.UTF-8
         - LANG=en_US.UTF-8
-        - IOS_SDK=iphonesimulator12.2
-        - IOS_VER=12.2 # Tests must run at the same OS version as $IOS_SDK
+#        - IOS_VER=$( xcrun --sdk iphonesimulator --show-sdk-version ) # Tests must run at the same OS version as $IOS_SDK
```
Comment (kovpas) [inline .travis.yml:34]: Please remove

**Comment by kovpas** [comment]
Thanks! Please fix one little nit and I'll merge this one in.

**Pull request merged**